redisraft.c

```diff
@@ -915,6 +915,16 @@ __attribute__((__unused__)) int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisMod
         return REDISMODULE_ERR;
     }
 
+    /* Sanity check that not running with cluster_enabled */
+    RedisModuleServerInfoData *info = RedisModule_GetServerInfo(ctx, "cluster");
+    const char * val = RedisModule_ServerInfoGetFieldC(info, "cluster_enabled");
+    int cluster_enabled = strncmp("1", val, 1);
+    RedisModule_FreeServerInfo(ctx, info);
+    if (!cluster_enabled) {
```

Comment:
```suggestion
    int cluster_enabled = (int) RedisModule_ServerInfoGetFieldSigned(info, "cluster_enabled", NULL);
    RedisModule_FreeServerInfo(ctx, info);
    if (cluster_enabled) {
```

A nitpick, sounds like `cluster_enabled` value should be reverted.

Reply:
well, before when dealing with as a string, it made sort of sense, but variable naming wasn't great for that, but treating it as a number now, everything is better.